plugins/k8saudit_eks/Makefile

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (C) 2022 The Falco Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+SHELL=/bin/bash -o pipefail
+GO ?= go
+
+NAME := k8saudit_eks
```

Comment:
I prefer k8saudit-eks. 

We prefer to use underscores only in source names. The dash is preferable for plugin names (I know, dummy_c is unfortunate 😅)

Reply:
@leogr I prefer the dash too, but I followed the current convention, only the underscore is allowed, see:
https://github.com/falcosecurity/plugins/blob/master/build/registry/pkg/registry/validate.go#L24

```go
var (
	rgxName   = regexp.MustCompile(`^[a-z]+[a-z0-9-_]*$`)
	rgxSource = regexp.MustCompile(`^[a-z]+[a-z0-9_]*$`)
)
```

Reply:
Ok, I know, I had to rebase on the upstream, in my locale version, the dash wasn't allows by the validate pkg, see:
```shell
❯ make check-registry
error: plugin name does follow the naming convention: 'k8saudit-eks'
```

After a rebase, it's ok.

Reply:
:+1: